src/main/java/com/ss/lms/entity/Author.java

```diff
@@ -1,11 +1,42 @@
 package com.ss.lms.entity;
 
+import java.io.Serializable;
 import java.util.List;
 
-public class Author {
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+@Entity
+@Table(name = "tbl_author")
+public class Author implements Serializable {
+
+	/**
+	 * 
+	 */
+	private static final long serialVersionUID = -5362668537632529686L;
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
 	private Integer authorId;
+
+	@Column(name = "authorName")
 	private String authorName;
-	//private List<Book> books; // many to many
+
+	@JsonIgnore
+	@ManyToMany(fetch = FetchType.EAGER)
```

Comment:
also do you want this fetch to be eager if you just gonna ignore the data during serialzation.
Why not just make it lazy and only worry about getting the data when you need it.